io/test/SConscript

```diff
@@ -28,50 +28,50 @@ event_manager_test = env.UnitTest('event_manager_test',
                                   ['event_manager_test.cc'],
                                  )
 
-env.Alias('src/io:event_manager_test', event_manager_test)
+env.Alias('io:event_manager_test', event_manager_test)
```

Comment:
Was this decision intentional? having just the `io:${tst_name}`

I was expecting `src/contrail-common/io:` ...